### ISSUE TITLE
Proposal: allow passing additional parameters to `@cypher`

### ISSUE THREAD

**Issue by wheresrhys** [neo4j/graphql-tracker-temp#21]
e.g. a use case for this might be that there is a `@cypher`-defined property that fetches a list of records. It would be useful to be able to add parameters that turn on filters of some sort to this query e.g. only records newer than X days old.

It seems to me that at the database and resolver level this would be fairly straightforward to implement, but more problematic when it comes to designing the directive and schema. I'm not gonna even try to propose what it might look like as I can't think of anything that isn't quite ugly/hacky.

**Comment by danstarns** [comment]
Hi @wheresrhys you can define field args on a cypher directive;
```
type User {
    additionalParameters(param: String!): String
        @cypher(
            statement: """
            RETURN $param
            """
        )
}
```
 Is this what you are looking for?

**Comment by wheresrhys** [comment]
Yes exactly that. Am I think I'm right in thinking this is new compared to neo4j-graphql-js? Easy to miss in the docs for the new library as the feature is used in one of the examples but not explicitly mentioned anywhere.

**Comment by danstarns** [comment]
This is already a feature and should be supported in both versions, let us know if you are having any issues with it. Thank you for pointing out the lack of docs, we are working on them currently, will take consideration.
